messages/md.deploy.json

```diff
@@ -25,7 +25,8 @@
     "verbose": "verbose output of deploy results",
     "validatedDeployRequestId": "request ID of the validated deployment to run a Quick Deploy",
     "singlePackage": "Indicates that the zip file points to a directory structure for a single package",
-    "soapDeploy": "deploy metadata with SOAP API instead of REST API"
+    "soapDeploy": "deploy metadata with SOAP API instead of REST API",
+    "purgeOnDelete": "the deleted components in the destructiveChanges.xml manifest file aren't stored in the Recycle Bin. Instead, they become immediately eligible for deletion."
```

Comment:
```suggestion
    "purgeOnDelete": "specify that deleted components in the destructive changes manifest file are immediately eligible for deletion rather than being stored in the Recycle Bin"
```